### DISCUSSION TITLE
help with software graph view

### DISCUSSION THREAD

**Discussion by linkaszelda** [artisan-roaster-scope/artisan#1129] [Q&A]
Hi there, I am roasting using this software and accidentally seemed to have cleared some settings my predecessor set up.

Ideally, when we load a profile it shows ET and BT in full view, without having to pan. Presently, it snaps to full view of only BT upon events such as start, charge, dry, etc.

How do I lock it in place in that fully zoomed out view I want it to have?

thanks 🙏

**Comment by bigl00z3** [comment]
Not totally sure if I understand what you mean. Maybe try uploading a screenshot.

I see a few possibilities:

- The first one I see is that your settings in `Config/Axes...` were modified. You could try ticking the `Auto` tickbox. That would show the full roast. You can also edit your axes values in this window:

<img width="689" alt="image" src="https://user-images.githubusercontent.com/8886922/226020223-7be20d75-afcf-4fb2-b447-bb04ed580a06.png">

- If one of the curve is missing, you may have disabled it by clicking either on one of the LCD or on the legend:

<img width="254" alt="image" src="https://user-images.githubusercontent.com/8886922/226021547-4d16f771-c507-4833-96ae-25f9f3370d1d.png">
or on 
<img width="95" alt="image" src="https://user-images.githubusercontent.com/8886922/226021674-df5a588c-26c7-42b1-9ec4-b15b0de38166.png">

if this doen't help, can you please upload a screenshot of your problem and your artisan-settings which you can export via `/Help/Save Settings... ` .
Note: to attach the file, do it on the GitHub page (not by email) and rename the `artisan-settings.aset `file with `.txt` extension at the end of the name for it to upload properly.
Nico

**Reply by linkaszelda** [reply]
Thanks for your reply!!
I've got to be honest, I'm a little afraid to touch it again because after a few hours of pressing things like "Switch Et < - > BT" and then "Switch Profiles", resetting my PC and the software, I've managed to get it back to working somewhat like it did before...

but I do have some pictures I took.
Here is how it would ideally display:
![20230317_161433](https://user-images.githubusercontent.com/97557764/226028225-e053acde-61f0-4cb3-876d-0e377ece4277.jpg)


This is how it was "snapping" before. By that I mean it would snap back to this view, zooming in on BT only. I was still able to pan and zoom enough to get it back to an approximation of the desired view, but upon events (such as Start, End, Charge, Etc) it would snap back to this view
![20230317_142051](https://user-images.githubusercontent.com/97557764/226028343-de9bb2b4-2a81-4429-853f-dbf629936a83.jpg)


should it happen again, I will try what you suggested. However, this is our current axes setting:
![20230317_161842](https://user-images.githubusercontent.com/97557764/226027734-cf75c750-76fc-4056-96da-9ce1a749fd0d.jpg)

**Reply by linkaszelda** [reply]
so sorry, swapped the photos. just edited!

**Reply by bigl00z3** [reply]
Well if it works fine now all good. 
But what I would recommend is to do a `Help/Save Settings...` and keep the `artisan-settings.aset` file backed-up somewhere safe (generally a good idea is to rename it with a date or version if you want to be sure to find the right one at a later stage). 

If you mess up your configuration you can always reload your previous "working" settings by going to `Help/Load Settings...` that will save you a lot of time and the risk of having to start from scratch.

Nico
